### DISCUSSION TITLE
Force Wii U Pro Controller to identify as Switch Pro Controller so Steam shows proper button glyphs.

### DISCUSSION THREAD

**Discussion by CheezyFriez12** [KeyPuncher/WiinUPro#148] [Ideas]
Not sure if this is possible but it would be awesome for us Big Picture Mode/SteamInput users.

**Comment by DraakjeYoblama** [comment]
That would be cool, but I doubt it will happen.

WiinUPro relies on 3rd party software for its virtual controllers, that software doesn't support virtual Switch pro controllers.

**Comment by KeyPuncher** [comment]
This would have to take place at the driver level which is beyond my scope of knowledge at the moment. I might know more about drivers in a few months but not specifically for Windows.
